wire-runtime/src/test/proto-kotlin/com/squareup/wire/protos/person/Person.kt

```diff
@@ -0,0 +1,115 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: person.proto
+package com.squareup.wire.protos.person
+
+import com.squareup.wire.EnumAdapter
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.WireEnum
+import com.squareup.wire.kotlin.UnkownFieldsBuilder
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.List
+import okio.ByteString
+
+data class Person(
+    val name: String,
+    val id: Int,
+    val email: String? = null,
+    val phone: List<PhoneNumber>,
+    val unknownFields: ByteString = ByteString.EMPTY
+) {
+    object ADAPTER : ProtoAdapter<Person>(FieldEncoding.LENGTH_DELIMITED, Person::class.java) {
+        override fun encodedSize(value: Person): Int = ProtoAdapter.STRING.encodedSizeWithTag(1, value.name) + 
+        ProtoAdapter.INT32.encodedSizeWithTag(2, value.id) + 
+        ProtoAdapter.STRING.encodedSizeWithTag(3, value.email) + 
+        PhoneNumber.ADAPTER.asRepeated().encodedSizeWithTag(4, value.phone) + 
+        value.unknownFields.size()
+
+        override fun encode(writer: ProtoWriter, value: Person) {
+            ProtoAdapter.STRING.encodeWithTag(writer, 1, value.name)
+            ProtoAdapter.INT32.encodeWithTag(writer, 2, value.id)
+            ProtoAdapter.STRING.encodeWithTag(writer, 3, value.email)
+            PhoneNumber.ADAPTER.asRepeated().encodeWithTag(writer, 4, value.phone)
+            writer.writeBytes(value.unknownFields)
+        }
+
+        override fun decode(reader: ProtoReader): Person {
+            var name: kotlin.String? = null
+            var id: kotlin.Int? = null
+            var email: kotlin.String? = null
+            var phone = mutableListOf<com.squareup.wire.protos.person.Person.PhoneNumber>()
```

Comment:
Are you missing a `%T` in the emitting code? This shouldn't be a FQCN.

Reply:
Kotlin types should get imported too